dagfactory/generators/airbyte_generator.py

```diff
@@ -251,15 +250,16 @@ def generate_tasks(self, params: Dict[str, Any]) -> Dict[str, Any]:
         if deploy_path:
             subprocess.run(["rm", "-rf", deploy_path], check=True)
 
+        manifest_json = json.load(open(os.path.join(cwd, "target", "manifest.json")))
```

Comment:
Use `pathlib` which is already imported

Reply:
Replaced os.path with Path(cwd)/target/manifest.json

84a49b5[](url)